AdyenDropIn/Utilities/ComponentManager+PaymentComponentBuilder.swift

```diff
@@ -156,6 +159,28 @@ extension ComponentManager: PaymentComponentBuilder {
                      context: context,
                      configuration: .init(style: configuration.style.formComponent))
     }
+    
+    internal func build(paymentMethod: CashAppPayPaymentMethod) -> PaymentComponent? {
+        #if canImport(PayKit)
+            guard var cashAppPayConfig = configuration.cashAppPay else {
+                AdyenAssertion.assertionFailure(
```

Comment:
Maybe `adyenPrint("Cash App Pay configuration instance must not be nil in order to use CashAppPayComponent")` ?
In case merchant is integrating with PayKit not via Adyen?

Reply:
Discussed with product and this is indeed fine